react/features/overlay/components/SuspendedOverlay.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+
+import AbstractOverlay from './AbstractOverlay';
+
+/**
+ * Implements a React Component for suspended overlay. Shown when a suspend is
+ * detected.
+ */
+export default class SuspendedOverlay extends AbstractOverlay {
+    /**
+     * Constructs overlay body with the message and a button to rejoin.
+     *
+     * @returns {ReactElement|null}
+     * @override
+     * @protected
+     */
+    _renderOverlayContent() {
+        const btnClass = 'inlay__button button-control button-control_primary';
+
+        /* eslint-disable react/jsx-handler-names */
```

Comment:
Do you know that `eslint-disable` will seize to be in effect as you expect? I'd rather we be safe rather than sorry and `eslint-enable` after a `eslint-disable`.